d2l-button-toggle-radio-group-item.js

```diff
@@ -0,0 +1,32 @@
+import {D2LButtonToggle} from './d2l-button-toggle.js';
+
+/**
+ * @customElement
+ * @polymer
+ */
+
+let subTemplate;
+class D2LButtonToggleRadioGroupItem extends D2LButtonToggle {
+	static get is() { return 'd2l-button-toggle-radio-group-item'; }
+
+	static get template() {
+		if (!subTemplate) {
+			subTemplate = super.template.cloneNode(true);
+			const button = subTemplate.content.querySelector('button');
+			button.setAttribute('aria-checked', '[[pressed]]');
+			button.setAttribute('aria-role', 'radio');
+			subTemplate.innerHTML = subTemplate.innerHTML.replace('aria-checked=', 'aria-checked$=');
+		}
+		return subTemplate;
+	}
+	static get properties() {
+		return {
+			value: {
+				type: String,
+				value: ''
```

Comment:
Shouldn't need to set a default of empty string -- it's empty by default already.